unit-tests/algo/depth-to-rgb-calibration/scene-data.h

```diff
@@ -108,6 +108,8 @@ struct scene_metadata
     uint64_t n_iterations;        // how many steps through optimization, and how many iteration file sets
     double correction_in_pixels;  // XY movement
     uint64_t n_edges;             // strong edges, i.e. after suppression
+    uint64_t valid_pixels;
```

Comment:
It's the "number of" pixel and not the pixels themselves... please denote this with "n_" here and for valid_ir_edges.